apps/console/src/features/oidc-scopes/pages/oidc-scopes.tsx

```diff
@@ -20,16 +20,17 @@ import { hasRequiredScopes } from "@wso2is/core/helpers";
 import { AlertLevels, TestableComponentInterface } from "@wso2is/core/models";
 import { addAlert } from "@wso2is/core/store";
 import { ListLayout, PageLayout, PrimaryButton } from "@wso2is/react-components";
-import { sortBy } from "lodash";
-import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { sortBy as _sortBy } from "lodash";
```

Comment:
Any reason for the dangling underscore. Also shall we use the following convention to aid tree shaking.
```suggestion
import sortBy from "lodash/sortBy";
```

Reply:
Resolved by 2525128